benchmark/bench/_util.js

```diff
@@ -1,6 +1,9 @@
-import { createRequire } from 'module';
+import { createRequire } from 'node:module';
+import path from 'node:path';
 
-export const astroBin = createRequire(import.meta.url).resolve('astro');
+const astroPkgPath = createRequire(import.meta.url).resolve('astro/package.json');
+
+export const astroBin = path.resolve(astroPkgPath, '../astro.js');
```

Comment:
Since the `"astro"` entrypoint doesn't point to the CLI anymore, I changed the heuristic to find the CLI file.